pkg/components/imguploader/azureblobuploader.go

```diff
@@ -45,7 +46,7 @@ func (az *AzureBlobUploader) Upload(ctx context.Context, imageDiskPath string) (
 	// setup client
 	blob := NewStorageClient(az.account_name, az.account_key)
 
-	file, err := os.Open(imageDiskPath)
+	file, err := os.Open(filepath.Clean(imageDiskPath))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.